tests/specs/controllers/OpenEAppController.test.js

```diff
@@ -148,4 +162,56 @@ describe('OpenEAppController', () => {
             );
         });
     });
+
+    describe('date countdown', () => {
+        beforeEach(async () => {
+            sandbox.stub(HelperService, 'getUserData').callsFake(() => ({
+                loggedIn: true,
+            }));
+            sandbox.stub(Application, 'find').resolves(resolvedAppData);
+            sandbox
+                .stub(OpenEAppController, '_getApplicationDataFromCasebook')
+                .resolves(resolvedCasebookData);
+        });
+
+        it('returns expired document error if days are negative', async () => {
+            // when
+            sandbox.stub(Date, 'now').callsFake(() => TWO_DAYS_AFTER_DEADLINE);
+            await OpenEAppController.renderPage(reqStub, resStub);
+
+            // then
+            const sailsErrorLogObj = sails.log.error.getCall(0).args[0];
+            expect(sailsErrorLogObj.message === 'Application has expired').to.be
+                .true;
+            expect(resStub.serverError.called).to.be.true;
+        });
+        it('shows correct number of days for 11 day old application', async () => {
```

Comment:
what happens when the deadline is in 1 hour? (0 days, 1hr?) - would you just show 0 days to download?

Reply:
Yep, currently, it's just 0 days but there's talk of changing the copy of some sort

Reply:
do you have i18n or a way of handling "1 day" vs "0 days" vs "20 days" etc?

Reply:
Hmmm, I don't have i18n but I don't follow what you mean by handling different days. Are you referring to different languages? Or something else?

Reply:
Yeah good shout, I think they do. But I don't know if it's worth downloading a whole library just for that one change 🤔

Reply:
It might not be, but maybe an if (or ternary) statement in the ejs file will suffice?